sysid-library/build.gradle

```diff
@@ -4,10 +4,11 @@ plugins {
 }
 
 // For using remote development artifacts
-wpi.maven.useLocal = false
-wpi.maven.useDevelopment = true
-wpi.versions.wpilibVersion = '2022.+'
-wpi.versions.wpimathVersion = '2022.+'
+// wpi.maven.useLocal = false
+// wpi.maven.useDevelopment = true
+// wpi.versions.wpilibVersion = '2022.+'
+// wpi.versions.wpimathVersion = '2022.+'
```

Comment:
Shouldn’t the 2022.+ pieces be kept?

Reply:
Yeah I had to add that in in order to get it to use the ADIS stuff on allwpilib